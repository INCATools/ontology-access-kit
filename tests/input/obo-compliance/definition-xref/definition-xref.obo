format-version: 1.4
ontology: definition-xref

[Term]
id: X:1
def: "x1 def" [PMID:123456]

