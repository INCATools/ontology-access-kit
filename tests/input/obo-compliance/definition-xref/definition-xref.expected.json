{
  "graphs" : [ {
    "id" : "http://purl.obolibrary.org/obo/definition-xref.owl",
    "meta" : {
      "basicPropertyValues" : [ {
        "pred" : "http://www.geneontology.org/formats/oboInOwl#hasOBOFormatVersion",
        "val" : "1.4"
      } ]
    },
    "nodes" : [ {
      "id" : "http://purl.obolibrary.org/obo/IAO_0000115",
      "lbl" : "definition",
      "type" : "PROPERTY"
    }, {
      "id" : "http://purl.obolibrary.org/obo/X_1",
      "type" : "CLASS",
      "meta" : {
        "definition" : {
          "val" : "x1 def",
          "xrefs" : [ "PMID:123456" ]
        }
      }
    }, {
      "id" : "http://www.geneontology.org/formats/oboInOwl#hasDbXref",
      "lbl" : "database_cross_reference",
      "type" : "PROPERTY"
    }, {
      "id" : "http://www.geneontology.org/formats/oboInOwl#hasOBOFormatVersion",
      "lbl" : "has_obo_format_version",
      "type" : "PROPERTY"
    } ]
  } ]
}